source/assets/js/backtotop.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Created by julchenrehme on 05.02.18.
+ */
+jQuery(document).ready(function($){
+    // browser window scroll (in pixels) after which the "back to top" link is shown
+    var offset = 300,
+        //browser window scroll (in pixels) after which the "back to top" link opacity is reduced
+        offset_opacity = 1200,
+        //duration of the top scrolling animation (in ms)
+        scroll_top_duration = 700,
+        //grab the "back to top" link
+        $back_to_top = $('.dev-top');
+
+    //hide or show the "back to top" link
+    $(window).scroll(function(){
```

Comment:
Please don't use event handler shortcuts instead use the `.on()` method to register a new event handler
```
$('window').on('scroll', function() {
   // ... your code
})
```

Reply:
check

Reply:
Unfortunately the recommendation doesn't seems to work the way you indented it, therefore I haven't changed the part. Otherwise the scrolling tracking isn't working properly and the "back to top" button wouldn't be visible.